tests/lib/prepare-project.sh

```diff
@@ -68,17 +68,18 @@ fi
 
 # declare the "quiet" wrapper
 . "$TESTSLIB/quiet.sh"
+. "$TESTSLIB/dirs.sh"
```

Comment:
It wont be exactly one in `spread.yaml` as we will have linode, qemu and external systems. So we will have this repeated at least three times. So we really need something like `environment-initializers`